force-app/main/default/lwc/groupServiceDelivery/groupServiceDelivery.js

```diff
@@ -0,0 +1,88 @@
+import { api, LightningElement, wire } from "lwc";
+import { getRecord } from "lightning/uiRecordApi";
+
+import SERVICE_DELIVERY_OBJECT from "@salesforce/schema/ServiceDelivery__c";
+import SERVICE_FIELD from "@salesforce/schema/ServiceDelivery__c.Service__c";
+import CONTACT_FIELD from "@salesforce/schema/ServiceDelivery__c.Contact__c";
+import PROGRAM_ENGAGEMENT_FIELD from "@salesforce/schema/ServiceDelivery__c.ProgramEngagement__c";
+import BULK_SERVICE_DELIVERY_FIELD_SET_FIELD from "@salesforce/schema/Service__c.BulkServiceDeliveryFieldSet__c";
+
+import getFieldSet from "@salesforce/apex/FieldSetController.getFieldSetForLWC";
+
+import selectService from "@salesforce/label/c.BSDT_Select_Service";
+
+const DEFAULT_FIELD_SET = "Bulk_Service_Deliveries";
+
+export default class GroupServiceDelivery extends LightningElement {
+    objectApiName = SERVICE_DELIVERY_OBJECT.objectApiName;
+    fieldSetName = DEFAULT_FIELD_SET;
+    serviceId;
+    fieldSet;
+
+    serviceField = SERVICE_FIELD.fieldApiName;
+    fieldSetField = BULK_SERVICE_DELIVERY_FIELD_SET_FIELD.fieldApiName;
+    fieldsToExclude = [
+        CONTACT_FIELD.fieldApiName,
+        PROGRAM_ENGAGEMENT_FIELD.fieldApiName,
+        this.serviceField,
+    ];
+    labels = { selectService };
+
+    @wire(getRecord, {
+        recordId: "$serviceId",
+        fields: [BULK_SERVICE_DELIVERY_FIELD_SET_FIELD],
+    })
+    wiredSession(result) {
```

Comment:
Curious about the method name.  Appears to get a getRecord against a service - would expect wiredService.

Reply:
copy and paste error lol... I will fix on the next iteration